index.html

```diff
@@ -2627,13 +2628,15 @@ <h3>Refreshing</h3>
         <dl>
           <dt><var>refreshService</var></dt>
           <dd>
-The value of the <code>refreshService</code> <a>property</a> MUST be one or
-more refresh services that provides enough information to the recipient's
-software such that the recipient can refresh the <a>verifiable credential</a>.
-Each <code>refreshService</code> value MUST specify its <code>type</code> (for
+The value of the <code>refreshService</code> <a>property</a> MUST be one or more
+refresh services that provides enough information to the recipient's software
+such that the recipient can refresh the <a>verifiable credential</a>. Each
+<code>refreshService</code> value MUST specify its <code>type</code> (for
 example, <code>ManualRefreshService2018</code>) and its <code>id</code>, which
-is the <a>URI</a> of the service. The precise content of each refresh service is
-determined by the specific <code>refreshService</code> <a>type</a> definition.
+is the <a>URI</a> of the service. There is an expectation that machine readable
+information needs to be retrievable from the URI. The precise content of
```

Comment:
I expect that we are going to be dropping this `id` requirement in the VC 2.0 work and replacing it with a `url` field (that may or may not point to machine readable information). Refresh services can be mediated (human in the loop) or unmediated (machine to machine), the `url` just identifies a place where you can initiate a protocol of some kind. That protocol might just be the software opening a URL to a page that a human then needs to interact with. The protocol might be something that can be automated.

Here's an example of a spec that is expected to end up in the VCWG in time that specifies a data model and protocol for this property.

https://digitalbazaar.github.io/vc-refresh-2021/

Reply:
FWIW, I think a specification of a ["ceremony"](https://eprint.iacr.org/2007/399.pdf) like this makes a lot of sense, where the 'url' field might be a [navigation](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigating-across-documents) target for a human participant rather than just a [fetch](https://fetch.spec.whatwg.org/#fetching) target for a machine.

Reply:
> where the 'url' field might be a [navigation](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigating-across-documents) target for a human participant rather than just a [fetch](https://fetch.spec.whatwg.org/#fetching) target for a machine.

Yep, and speaking specifically to "navigation target for a human participant", one place that's being defined is here:

https://w3c-ccg.github.io/vp-request-spec/#interaction-types

When a Verifiable Credential Verifier wants something from you, they send you (the Holder) a Verifiable Presentation Request. That Verifiable Presentation Request has an `interact` field, which can instruct software on how it should handle the interaction. Sometimes, that might be by opening a web browser and taking the individual to a website to do some manual process (and they get their Verifiable Credential at the end of that process). Other times, like in the case of vc-refresh-2021 (where refresh can happen automatically), that might be executing some sort of machine-to-machine protocol.

The VC Refresh 2021 spec is just one spec that uses this interaction approach using the Verifiable Credentials API, and the thing that kicks all of that off is the `refreshService` in the Verifiable Credential.

In any case, more detail on your suggestion -- yes, we're going down that path with at least one of the VC protocols.